linkerd/app/core/src/metrics/tcp_accept_errors.rs

```diff
@@ -21,7 +21,7 @@ metrics::metrics! {
 
 #[derive(Clone, Debug)]
 pub struct Registry {
-    scopes: metrics::SharedStore<TargetAddr, Scope>,
+    scopes: metrics::SharedStore<OrigDstAddr, Scope>,
```

Comment:
Not wanting to just key the registry on `SocketAddr`s was the original reason I didn't do this and just wrap the values in the type that knows how to format labels when actually formatting them. But, at the time, we were only including the target port, that was more important. Now that the label includes the entire address, this is probably fine.

Personally, I thought it was conceptually nicer to have the metrics registry be indexed by the label type rather than by a non-metrics-specific label type...but this definitely reduces the complexity of all the other code!